src/content/en/tools/lighthouse/audits/tap-targets.md

```diff
@@ -0,0 +1,68 @@
+project_path: /web/tools/_project.yaml
+book_path: /web/tools/_book.yaml
+description: How to pass the "Tap targets are not sized appropriately" Lighthouse audit.
+
+{# wf_updated_on: 2019-02-15 #}
+{# wf_published_on: 2019-02-15 #}
+{# wf_blink_components: Platform>DevTools #}
+
+# Tap targets are not sized appropriately {: .page-title }
+
+## Overview {: #overview }
+
+[webmasters]: https://webmasters.googleblog.com/2015/04/rolling-out-mobile-friendly-update.html
+[MDN]: https://developer.mozilla.org/en-US/docs/Web/API/Element/getBoundingClientRect#Value
+
+Tap targets are interactive elements, like buttons or links, that users frequently tap.
+Appropriately-sized tap targets make pages more mobile-friendly and accessible. Google Search
+started boosting the ranking of mobile-friendly pages on mobile search results back in 2015.
+See [Rolling out the mobile-friendly update][webmasters].
+
+A tap target is inappropriately-sized when its bounding rectangle is less than 48 pixels. 48
```

Comment:
I'm not sure how much of this info is worth putting into the docs, but fyi here's a more detailed description of when we fail a tap target:

- targets that are at least 48x48px never fail
- we have some tolerance as well, so in practice tap targets greater than maybe 40x40px never fail
- links in a block of text never fail
- if a tap target is small AND there's another tap target nearby, then we fail the tap target

We never directly look at margins, instead we simulate a 48x48px finger and see if it touches anything it shouldn't touch. (If there is a margin that makes it less likely that the finger touches something it shouldn't.)

Reply:
I think we should point to remedies like adding margin or making the tap target bigger, and outline the success conditions we already talk about elsewhere in the docs (e.g. 48x48px min size). Rather than trying to explain what causes an LH failure.